### PR TITLE
fix(copy): Sign into to Sign in to

### PR DESCRIPTION
**Pull request by marcosmartini**
- Update occurrences of "sign into" to "sign in to"

## Summary by CodeRabbit

- **New Features**
	- Enhanced documentation for various SDKs, including organization management and authentication processes.
	- Added new sections and examples for improved clarity on using the Kinde SDK across different platforms (Elixir, Java, Next.js, etc.).
  
- **Bug Fixes**
	- Corrected grammatical errors and improved phrasing for clarity in multiple documentation files.

- **Documentation**
	- Streamlined installation and configuration instructions across SDKs.
	- Expanded sections on managing user permissions, feature flags, and organization-specific authentication.

- **Chores**
	- Minor formatting adjustments made throughout the documentation for consistency and readability.

### CODE REVIEW

File: src/content/docs/developer-tools/sdks/backend/nodejs-sdk.mdx [hunk 1]
```diff
@@ -232,7 +232,7 @@ You can also pass `org_name` as part of the query string as per the following HT
 The Kinde client provides methods for you easily sign up and sign in users into organizations. You can add links in your HTML as follows:
 
 ```html
-<a href="/login?org_code=<your_org_code>">Sign into an org</a>
```
Comment (clairekinde11) [inline src/content/docs/developer-tools/sdks/backend/nodejs-sdk.mdx:235]: Arguably this one is correct both ways

File: src/content/docs/developer-tools/sdks/backend/nextjs-sdkv1.mdx [hunk 2]
```diff
@@ -478,7 +478,7 @@ import {RegisterLink} from "@kinde-oss/kinde-auth-nextjs/server";
 This code should also be passed along with the `<LoginLink>` component if you wish for a user to be logged into a specific organization.
 
 ```jsx
-<LoginLink orgCode="org_123456">Sign into org</LoginLink>
```
Comment (clairekinde11) [inline src/content/docs/developer-tools/sdks/backend/nextjs-sdkv1.mdx:481]: Could be either for the button situation

File: src/content/docs/developer-tools/sdks/backend/remix-sdk.mdx [hunk 3]
```diff
@@ -76,7 +76,7 @@ import { Link } from "@remix-run/react";
 </Link>
 ```
 
-**Sign into organizations**
```
Comment (clairekinde11) [inline src/content/docs/developer-tools/sdks/backend/remix-sdk.mdx:79]: Either / or

File: src/content/docs/developer-tools/sdks/backend/typescript-sdk.mdx [hunk 4]
```diff
@@ -435,7 +435,7 @@ const registerUrl = await kindeClient.register(sessionManager, {org_code: "org_1
 
 ```typescript
 // HTML
-<button onclick="handleLoginIntoOrgnization('org_1234')">Sign into</button>
```
Comment (clairekinde11) [inline src/content/docs/developer-tools/sdks/backend/typescript-sdk.mdx:438]: I think this one should be "Sign in"

File: src/content/docs/developer-tools/sdks/backend/typescript-sdk.mdx [hunk 5]
```diff
@@ -435,7 +435,7 @@ const registerUrl = await kindeClient.register(sessionManager, {org_code: "org_1
...
+<button onclick="handleLoginIntoOrgnization('org_1234')">Sign in to</button>
```
Comment (clairekinde11) [inline src/content/docs/developer-tools/sdks/backend/typescript-sdk.mdx:438]: what code rabbit said - handleLogIntoOrganization

File: src/content/docs/developer-tools/sdks/native/flutter-sdk.mdx [hunk 6]
```diff
@@ -450,7 +450,7 @@ The `id_token` will also contain an array of organizations that a user belongs t
 
 You can also fetch the org_code by:
 
-1. Sign into [kinde.com](https://kinde.com/).
```
Comment (clairekinde11) [inline src/content/docs/developer-tools/sdks/native/flutter-sdk.mdx:453]: Into is better here

File: src/content/docs/integrate/third-party-tools/cloudflare-zero-trust.mdx [hunk 7]
```diff
@@ -16,7 +16,7 @@ You need to already have a backend [web application set up](/build/applications/
 
 ## Get your Cloudflare team domain
 
-1. [Sign into Cloudflare](https://dash.cloudflare.com/) and navigate to **Zero Trust**.
```
Comment (clairekinde11) [inline src/content/docs/integrate/third-party-tools/cloudflare-zero-trust.mdx:19]: Into is better here

**Review by clairekinde11 — CHANGES_REQUESTED** [review]
This is one of those times a global change only works 90% of the time. In most cases 'in to' is correct, in some cases 'into' is better.

**Comment by marcosmartini** [comment]
@clairekinde11 , I'm done with the updates.

**Review by clairekinde11 — APPROVED** [review]

**Pull request merged**